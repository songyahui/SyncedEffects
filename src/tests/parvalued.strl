module parvalued:

output J : integer;

signal I : integer in
       present I then emit J(?I) end || emit I(5)
end signal

end module