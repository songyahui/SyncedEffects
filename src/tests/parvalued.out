parvalued> ;
--- Output: J(5)
parvalued> ;
--- Output:
parvalued> ;
--- Output:
parvalued> ;
--- Output:
